rust/worker/src/server.rs

```diff
@@ -143,14 +147,12 @@ impl WorkerServer {
         let collection_uuid = CollectionUuid::from_str(&collection.id)
             .map_err(|_| Status::invalid_argument("Invalid Collection UUID"))?;
 
-        let vector_uuid = SegmentUuid::from_str(&scan.knn_id)
-            .map_err(|_| Status::invalid_argument("Invalid UUID for Vector segment"))?;
-
-        let metadata_uuid = SegmentUuid::from_str(&scan.metadata_id)
-            .map_err(|_| Status::invalid_argument("Invalid UUID for Metadata segment"))?;
-
-        let record_uuid = SegmentUuid::from_str(&scan.record_id)
-            .map_err(|_| Status::invalid_argument("Invalid UUID for Record segment"))?;
+        let metadata_uuid =
```

Comment:
Please add a comment here on the `unwrap_or` behavior

Reply:
This is a temporary impl that will be removed in the next PR